packages/app/src/vis-packs/core/line/LineVisContainer.tsx

```diff
@@ -53,6 +56,7 @@ function LineVisContainer(props: VisContainerProps) {
               title={entity.name}
               toolbarContainer={toolbarContainer}
               config={config}
+              invalidValue={fillValue}
```

Comment:
I have then to do a bit a prop drilling to use this `fillValue` in the domain computation of `MappedLineVis` and in the generation of points in the `DataCurve`.